smartcar_frontend/src/pages/home/Home.js

```diff
@@ -29,28 +29,30 @@ class HomeComponent extends React.Component {
         </header>
         <NavLink className="active" to="/race">
           <div className="Home-link">
-            <li className="Home-linkItem">Race</li>
+          <li className="Home-linkItem">Race</li>
           </div>
         </NavLink>
         <NavLink className="active" to="/LeaderBoard">
           <div className="Home-link">
-            <li className="Home-linkItem">Leaderboard</li>
+          <li className="Home-linkItem">Leaderboard</li>
           </div>
         </NavLink>
         <NavLink className="active" to="/race_times">
           <div className="Home-link">
-            <li className="Home-linkItem">Race times</li>
+          <li className="Home-linkItem">Race times</li>
           </div>
         </NavLink>
-        <li className="Difficulty" onClick={this.onClick}>
-          Difficulty : {this.state.difficultyLevel}
-          {this.state.difficulty ? (
-            <Difficulty onSelectDiff={this.handleDifficulty} />
-          ) : null}
-        </li>
+            <select className="DifficultyList" name="Difficulty" id="difficulties">
```

Comment:
Have to add the functionality to the state. Have created a separate issue for this.